docs/walkthrough/walkthrough-kind.md

```diff
@@ -183,6 +190,9 @@ kubectl wait -n tekton-pipelines \
 
 The Tekton Dashboard can import resources directly from a GitHub repository.
 
+Below steps will help you to understand how to import an example pipelinerun file [output-pipelinerun.yaml](https://github.com/tektoncd/pipeline/blob/master/examples/v1beta1/pipelineruns/output-pipelinerun.yaml)
+The dashboard will import this pipelinerun resources, which has a pipeline and that pipeline has some tasks that executes some steps sequentially. To learn more about how the pipelinerun works you can take a look at the example file [output-pipelinerun.yaml](https://github.com/tektoncd/pipeline/blob/master/examples/v1beta1/pipelineruns/output-pipelinerun.yaml).
```

Comment:
We can simplify this a bit and avoid duplicating the link. Also where possible we should provide meaningful text as the content of the link instead of using the raw URL.

```suggestion
The following steps demonstrate how to import a set of resources from a file hosted in a GitHub repository and view the resulting `PipelineRun`. The example contains some `Tasks`, a `Pipeline` that uses them, and a `PipelineRun` to execute it. To learn more about what the example does, take a look at the [example file](https://github.com/tektoncd/pipeline/blob/master/examples/v1beta1/pipelineruns/output-pipelinerun.yaml) we'll be using.
```